models/applications.ts

```diff
@@ -1,6 +1,56 @@
 import { application } from "../types/application";
 const firestore = require("../utils/firestore");
 const ApplicationsModel = firestore.collection("applicants");
+const { DOCUMENT_WRITE_SIZE: FIRESTORE_BATCH_OPERATIONS_LIMIT } = require("../constants/constants");
+import { chunks } from "../utils/array";
+
+const batchUpdateApplications = async () => {
+  try {
+    const operationStats = {
+      failedApplicationUpdateIds: [],
+      totalFailedApplicationUpdates: 0,
+      totalApplicationUpdates: 0,
+    };
+
+    const updatedApplications = [];
+    const applications = await ApplicationsModel.get();
```

Comment:
Hey Vinayak, can you share some context about this application model, what it does?

Reply:
so any one who wants to join in RDS needs to fill in a form, which gets stored in applicants collection. and the super user can see all those request, accept or reject it.

Reply:
Are we getting all the entries? shouldn't they be on some condition like `where createdAt = null`?

Reply:
firebase doesn't allow that, so if I filter the applications with where statement containing createdAt, it will ignore those docs.

Reply:
Does that filtering requires an existing index on that columns?